src/components/ChatEntry.css

```diff
@@ -1,59 +1,3 @@
-button {
```

Comment:
I'm seeing a number of changes in the CSS files, did you run into any issues with applying the styles?